src/index.js

```diff
@@ -0,0 +1,367 @@
+            onClick={(i) => this.squareOnClick(i)}
+            winnerLine={winnerLine}
+          />
+        </div>
+        <div className="game-info">
+          <div className="game-status">{status}</div>
+          <GameModeBtm
+            active={!this.state.stepNumber ? 'active' : ''}
+            text={this.state.playWithComputer ? 'friend' : 'computer'}
+            onClick={() => {
+              if (!this.state.stepNumber) {
+                this.setState({playWithComputer: !this.state.playWithComputer})}
+              }
+            }
+          />
+          <FirstStepRight
+            lock={!this.state.stepNumber ? '' : 'lock'}
+            hide={this.state.playWithComputer ? '' : 'hide'}
+            manFirst={() => {
+              this.resetState();
+              this.setState({
+                playerGoesFirst: true,
+                xIsNext: true,
+              })
+            }}
+            computerFirst={() => {
+              this.resetState();
+
+              const squares = Array(9).fill(null);
+              const squareNumber = Logic.firstStepAlgorithm(squares, false);
+              squares[squareNumber] = 'X';
+              this.setState({
+                history: [{
+                  squares: squares,
+                }],
+                playerGoesFirst: false,
+                xIsNext: false,
+              });
+            }}
+          />
+          <button
+            className={`game-restart ${!this.state.stepNumber ? 'hide' : ''}`}
+            onClick={(e) => {
+                this.resetState();
+
+                const btn = document.querySelector('.game-mode');
+                btn.classList.add('active');
+
+                const firstStepBtns = document.querySelectorAll('.first-step button');
+                firstStepBtns.forEach(btn => btn.classList.remove('active'));
+              }
+            }
+          >Start new game
+          </button>
+          <button
+            className={`game-reverse ${!this.state.stepNumber ? 'hide' : ''}`}
+            onClick={() => this.setState({reverseListOfMoves: !this.state.reverseListOfMoves})}
+          >Reverse list
+          </button>
+          <ol className={`game-list ${!this.state.stepNumber ? 'hide' : ''}`}>
+            <li className="game-list-toggle">
+              <button
+                onClick={(e) => {
+                  const list = e.currentTarget.closest('ol');
+
+                  if (list.classList.contains('open')) {
+                    list.classList.remove('open');
+                  } else {
+                    list.classList.add('open');
+                  }
+                }}
+              >
+                List of Moves
+              </button>
+            </li>
+            {moves}
+          </ol>
+        </div>
+      </div>
+    );
+  }
+}
+
+class Board extends React.Component {
+  renderSquare(i, className) {
+    return (
+      <Square
+        className={className || 'square'}
```

Comment:
you could use `default` parameter instead of this

Reply:
Thanks, I fixed it)